myapp/config/locales/model.ja.yml

```diff
@@ -0,0 +1,17 @@
+ja:
+  activerecord:
+    models:
+      # view側： User.model_name.human => "ユーザ" / t("activerecord.models.user")と同じ
+      task: タスク 
+    # model毎に定義したいattributesを記述
+    attributes:
+        task:
+          id: ID
+          # view側： User.human_attribute_name :name => "名前" /　t("activerecord.attributes.user.name")と同じ
+          title: タイトル
+          description: 詳細
+
+  # 全てのmodelで共通して使用するattributesを定義
+  attributes:
+    created_at: 作成日
+    updated_at: 更新日
```

Comment:
POSIX compliance requires a new line at the end of the file!

Reply:
Thank you, I add a new line.